### PR TITLE
feat(crux-ui): no data to display notices

### PR DESCRIPTION
**Pull request by bvektor**
When there is no data to display in a table/list we show a notice instead of an empty table/nothing.

### CODE REVIEW

File: web/crux-ui/locales/en/audit.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
-    "event": "Event",
-    "info": "Info"
+  "event": "Event",
+  "info": "Info",
+  "noItems": "Sorry, there's no activity which we could've logged. Add a registry to start onboarding."
```
Comment (polaroi8d) [inline web/crux-ui/locales/en/audit.json:4]: Registy in Audit logs?

**Review by polaroi8d — CHANGES_REQUESTED** [review]
Could you add a default 1 week filter in the audit log?

File: web/crux-ui/locales/en/deployments.json [hunk 2]
```diff
@@ -3,5 +3,6 @@
   "deploysNameDeploy": "Products - {{product}} - Versions - {{version}} - Deployments - {{name}} - Deploy",
   "log": "Log",
   "environmentConfig": "Environment config",
-  "created": "Created"
+  "created": "Created",
+  "noItems": "You haven't made a deployment. To add a deployment, click 'Deploy'."
```
Comment (robot9706) [inline web/crux-ui/locales/en/deployments.json:7]: I don't think there's a deploy button on the deployments page.
image

**Review by robot9706 — CHANGES_REQUESTED** [review]

**Review by robot9706 — APPROVED** [review]
😎

File: web/crux-ui/src/pages/audit-log.tsx [hunk 3]
```diff
@@ -32,8 +32,8 @@ const AuditLogPage = (props: AuditLogPageProps) => {
 
   const { auditLog } = props
 
-  const [startDate, setStartDate] = useState<Date>(null)
-  const [endDate, setEndDate] = useState<Date>(null)
+  const [startDate, setStartDate] = useState<Date>(new Date(Date.now() - 1000 * 3600 * 24 * 6))
```
Comment (polaroi8d) [inline web/crux-ui/src/pages/audit-log.tsx:35]: Please outsorce this "magic" number.

**Review by polaroi8d — APPROVED** [review]

**Pull request merged**